api/handler/response.go

```diff
@@ -17,3 +17,12 @@ type planIdResponse struct {
 	// Example: "456def"
 	Id string `json:"id"`
 }
+
+// sspIdResponse is a struct that holds the ID of a SSP.
+// swagger:model
+type sspIdResponse struct {
```

Comment:
You can remove this one and use idResponse struct

Reply:
There is no idResponse struct defined. Did you mean to create idResponse struct to replace all 3 (catalog,plan,ssp) defined here currently because they all look the same?

Reply:
Your branch is 17 commits behind main. You should update 😊